### PR TITLE
Create spanish translation of appendices

### PR DESCRIPTION
**Pull request by csralvall**
Spanish translation for appendices A0, A1, A2 and A3.

### CODE REVIEW

File: es/A3_0_Usando_Bitcoin_Regtest.md [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+En regtest, usualmente no hay suficientes transacciones por lo que bitcoind no
+puede darle un estimado confiable y, sin eso, el monedero no creara
+transacciones a menos que se configure explícitamente la tarifa.
+
+### Reconfigure la cadena de bloques Regtest
+
+Si usted lo desea, puede mas tarde reiniciar su Regtest con una nueva cadena de
+bloques.
+
+Los monederos Regtest y el estado de la cadena de bloques (estado de la cadena)
```
[the remark this answers is not in the document]
Comment (csralvall) [reply es/A3_0_Usando_Bitcoin_Regtest.md:38]: Yes. In spanish `blockchain state` and `chainstate` collide in their meaning.

**Comment by csralvall** [comment]
@javiervargas. Do you think that this is ready now?

**Review by javiervargas — APPROVED** [review]
Ready!,  thanks

**Review by javiervargas — APPROVED** [review]
@shannona ready to merge

**Pull request merged**